torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18869,12 +18869,6 @@ def __init__(
     PythonRefInfo(
         "_refs.nn.functional.softplus",
         torch_opinfo_name="nn.functional.softplus",
-        decorators=(
-            DecorateInfo(toleranceOverride({
-                torch.bfloat16: tol(atol=1e-2, rtol=0),
-                torch.float16: tol(atol=1e-3, rtol=0),
-            }), 'TestCommon', 'test_python_reference_consistency', device_type='cpu'),
```

Comment:
Looks like the skip is for `CPU`. Looking at the OpInfo, seems like it won't require an update. Thanks for looking at this.

Reply:
Ah, right. Will undo it. Thanks, Kshitij!